doc/tutorial/Server.lhs

```diff
@@ -149,7 +149,7 @@ repo](http://github.com/haskell-servant/servant) and look at the
 *doc/tutorial* directory. This code (the source of this web page) is in
 *doc/tutorial/Server.lhs*.
 
-If you run it, you can go to `http://localhost:8081/users` in your browser or
+If you run it (using `ghci -pgmL markdown-unlit` or `stack exec -- ghci -pgmL markdown-unlit` and then evaluating `main` into the repl), you can go to `http://localhost:8081/users` in your browser or
```

Comment:
How to run with `cabal new-run` ?

Reply:
what would be the complete command ?

Reply:
maybe some instruction in the `examples.md` above and/or in the `lhs` files could be useful to run the literate files.

I doubt newcomers would come up with the idea of running `stack exec -- ghci -pgmL markdown-unlit`  by themselves..

Reply:
I'd change `tutorial.cabal` so there would be a `executable tutorial-server`, so people can just `cabal new-run tutorial-server`.

Reply:
as of today, after a binary dependencies installation (say `hsc2hs`), `cabal new-repl doc/tutorial/` seem to load the tutorial correctly